napari/_vispy/vispy_image_layer.py

```diff
@@ -13,18 +14,24 @@
     np.dtype(np.int16),
     np.dtype(np.uint16),
     np.dtype(np.float32),
+    np.dtype(np.complex64),
 ]
 
 
 class VispyImageLayer(VispyBaseLayer):
     def __init__(self, layer):
-        self._image_node = ImageNode(None, method='auto')
+        self._image_node = (
+            ImageNode(None, method='auto')
+            if not layer.is_complex
+            else ComplexImageNode(None, method='auto')
+        )
```

Comment:
Out of curiosity, did you consider a way to do this such that we always use `ComplexImageNode` and that then can understand if data is complex or not rather than us making the choice here? Just thinking if we could push `ComplexImageNode` upstream one day so it just became `ImageNode` (in a similar way to how for `rgb` data we don't have `RGBImageNode` we just have `ImageNode`)